Sources/PointFree/Home.swift

```diff
@@ -44,10 +44,10 @@ private let episodeRowView = View<Episode> { ep in
       gridColumn(sizes: [.mobile: 12, .desktop: 7], episodeInfoColumnView.view(ep)),
 
       gridColumn(sizes: [.mobile: 12, .desktop: 5], [`class`([Class.grid.first(.mobile), Class.grid.last(.desktop)])], [
-        div([`class`([Class.size.height100pct]), style(lineHeight(0))], [
+        div([`class`([Class.size.height100pct]), style(lineHeight(0) <> gradient <> minHeight(.px(300)))], [
```

Comment:
have to do the `min-height` cause without an image the div collapses in mobile with no image.

Reply:
Wonder if we can set an aspect ratio to prevent glitches in general.

Reply:
I tried a few approaches towards that, none of em worked great